README.txt

```diff
@@ -39,28 +39,11 @@ following software:
 INSTALLATION:
 
 Instructions for installing Helioviewer.org are available online at
-http://wiki.helioviewer.org/wiki/Installing_Helioviewer.org
+http://wiki.helioviewer.org/wiki/index.php?title=Installing_Helioviewer.org
```

Comment:
We should fix these links, I'll update them with the github.io pages.

Reply:
This one, right? https://helioviewer-project.github.io/install/

Reply:
yep